src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -210,7 +238,15 @@ private List<string> ResolveByXRefMaps(List<string> uidList, ConcurrentDictionar
                 }
                 else
                 {
-                    list.Add(uid);
+                    if(XRefServerUrls != null && XRefServerUrls.Length > 0) spec = FindByRequest(uid);
```

Comment:
wrap by `{}`

---
In reply to: [125905169](https://github.com/dotnet/docfx/pull/1837#discussion_r125905169) [](ancestors = 125905169)